portfolio/src/main/webapp/style.css

```diff
@@ -1,93 +1,179 @@
-
 #content {
     margin-left: auto;
     margin-right: auto;
-    width: 650px;
     font-family: sans-serif;
     display: block;
     text-align: center;
     padding: 10px;
+    background-color: #F0F5F9;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
-/*make portrait image round*/
+
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #1E2022;
+    overflow: hidden;
+  }
+  
+/* Style the links inside the navigation bar */
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+  
+/* Change the color of links on hover */
+.topnav a:hover {
+    background-color: #C9D6DF;
+    color: black;
+}
+  
+  /* Add a color to the active/current link */
+.topnav a.active {
+    background-color: #52616B;
+    color: #F0F5F9;
+}
+
+/* Center text*/
+.name{
+    margin: auto;
+    align-items:center;
+    width:750px;
+    margin-top: 50px;
+    padding: 50px;
+}
+
+/* Rounded Headshot image*/
 img.rounded{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
         border-radius: 50%;
-        width: 300px;
+        width: 200px;
         height: auto;
-        margin-top: 35px;
-}
-
-/*make artist images round*/
-img.rounded-corners {
-    border-radius: 50%;
-    padding: 70px;
 }
 
-/*create box for about me class*/
-.aboutme {
-    
-    background-color: #c9d6df;
-    height: 1175px;
-    margin-right: auto;
+/*Centered text*/
+.about {
+    margin-top: 40px;
     margin-left: auto;
-    font-family: sans-serif;
-    display: block;
+    margin-right: auto;
+    align-items:center;
+    width: 750px;
     text-align: center;
     padding: 10px;
 }
 
-h1 {
-    padding: 10px;
-    font-size: 50px;
-    text-align: center;
-}
-h2 {
-    padding: 10px;
-    font-size: 30px;
-    text-align: center;
+/*Centered header for blog*/
+.blogheader{
+    margin: auto;
+    margin-top: 400px;
 }
 
-h3{
-    padding: 10px;
-    font-size: 30px;
+/*Create blog post*/
+.blog{
+    margin: auto;
+    width: 700px;
+    display: block;
     text-align: left;
+    border-radius: 25px;
+    background-color: #c9d6dF; 
+    padding: 10px;   
+}
+/* Centered contact header*/
+.contact{
+    margin: auto;
+    margin-top: 80px;
 }
 
-/*rules for paragraphs*/
-p{
-    margin-left: auto;
-    margin-right: auto;
-    width: 650px;
-    font-family: sans-serif;
-    display: block;
-    text-align: center;
+/* Adjust size and color on icons*/
+a{
```

Comment:
This will apply to everything that has `<a>` tag. I would suggest having a separate css class for icons.